ui-grid/ui-grid.d.ts

```diff
@@ -484,6 +484,10 @@ declare module uiGrid {
          */
         appScope?: ng.IScope;
         /**
+        * returns an array of columns in the grid
+        */
+        columns: Array<IGridColumn>;
```

Comment:
Fair enough!  @vvakame , ready to merge! :+1: 

@ashwin027 thanks for your contribution!